spec/acceptance/resource_languages_controller_spec.rb

```diff
@@ -139,6 +139,8 @@
         }
       }
     end
+    let(:language2) { Language.second }
+    let!(:attribute) { FactoryBot.create(:language_attribute, language: language2, resource: resource, key: "enable_tips", value: "false") }
```

Comment:
New data in an acceptance (integration) spec isn't really the best way to test a validation. For example, if I came back to this spec in a different context, I might be tempted to remove these lines as they don't seem to be doing anything. I think it would be nice to write something out in the LanguageAttribute model spec if we want to test something. But in general I don't usually write/check-in specs for framework tools like simple validations. I would approve this PR without a spec change, or help write a simple model test if you would be more comfortable. I'll defer to you, @frett.

Reply:
k, let me see if I can piece together a model spec